EMS/core-bundle/src/Controller/ContentManagement/EnvironmentController.php

```diff
@@ -432,6 +435,13 @@ public function addAction(Request $request): Response
                 } else {
                     $environment->setAlias($this->instanceId.$environment->getName());
                     $environment->setManaged(true);
+
+                    $length = $this->environmentRepository->counter();
+                    do {
+                        $newOrderKey = ++$length;
+                    } while ($this->environmentRepository->orderKeyExists($newOrderKey));
+                    $environment->setOrderKey($newOrderKey);
```

Comment:
There is maybe a better approach to get the max value from an sql query? Like `select max(environment.orderKey) from environment` ?